packages/types/src/chain.ts

```diff
@@ -32,7 +32,7 @@ export interface ChainMetadata {
   img: string;
   uniswapV3Routers?: {
     [key: string]: {
-      [key: string]: string;
+      [key: string]: string | null;
```

Comment:
Why?

Reply:
because the value is not hardcoded 
![image](https://github.com/ionicprotocol/monorepo/assets/2662912/5627e88b-b8a8-4992-9772-80d1f57d091f)